### ISSUE TITLE
[Bug Report]: Double checked everything but PS4/PS5 controllers still won't connect

### ISSUE THREAD

**Issue by jebediah2** [ndeadly/MissionControl#768]
### Switch Firmware Version

17.0.0 (Latest)

### Atmosphère Version

Other (please specify in issue description)

### Mission Control Version

0.10.0 (Latest)

### Boot Method

Hekate

### Issue Description

Everything is the latest version, including Atmosphère 1.6.2. I installed, re-installed, and re-re-installed MissionControl, have reset my official PS4 and PS5 controllers several times, but none of them work. I know my PS4 controller used to work some versions ago, and that's the first time I try to connect it with latest stuff.

I found some traces of [sys-con](https://github.com/cathery/sys-con), the old sysmodule that made some popular USB controllers work, and deleted everything.

Every time I edit the SD card content, I do an actual reboot using the 'Reboot' option in Hekate. I follow all instructions to the letter, but my PS4/PS5 controllers never get recognised in the 'Change Grip/Order' menu.

On Reddit I've seen people report that PS4 controllers often take a long time or several attempts before they eventually connect, but I've been patient doing that too.

I'm writing here looking for ideas and/or to see whether anyone else is having trouble on the latest versions of MC, Atmo and NX.

**Comment by jebediah2** [comment]
Apparently MissionControl doesn't even load. I checked by pressing DPAD_UP + MINUS and no screenshot is being taken. Unless that trick is outdated, this would indicate something is preventing MissionControl from loading in the first place.

I am transferring files from a Mac using Hekate's SD Card mounting feature, which has always worked for me. I have also done the same again by using an SD card adapter on the Mac, just in case. I have also cleared all possible archive bits using Hekate's dedicated feature, as well as running the following in Terminal:
`sudo chflags -R arch /Volumes/SDCARDNAME`

I am again running out of ideas what else I could do.

**Comment by jebediah2** [comment]
It isn't mentioned in the FAQ but the shortcut `DPAD_UP + MINUS` is only supposed to work with _detached_ joycons, not the opposite. So this troubleshooting step was never relevant to me after all.

While I was continuing my effort to clean up and re-re-install stuff on the SD card, I started having a sigpatches issue and did even more tinkering to solve it. At the point it was gone, I had apparently also solved the MissionControl problem because it started working without any issues. Therefore I cannot offer much details about how I actually did solve the issue, but here is a list of things I've done in the whole process:
- Delete all remnants of [sys-con](https://github.com/cathery/sys-con)
- Delete all MissionControl components, including backup folders I had kept, and put a fresh copy _from a Windows computer_.
- Re-install Atmosphère 1.6.2 and desired hacking components
- Boot with Atmosphère's latest fusee.bin
- Boot with latest Hekate (and from there the latest fusee.bin as a payload)
- Edit Hekate's hekate_ipl.ini to add `payload=bootloader/payloads/fusee.bin` in the SysNand CFW section
- After all this was done, I cleaned up the SD card with [BlueHarvest](https://www.zeroonetwenty.com/blueharvest/) (I always do this step as a precaution)
- Finally, I ran Hekate's archive bit fixer again

Like I said, there is no way to tell which among these steps did the trick, especially considering I had done most of them at least once already, but that's all I had to do to fix the problem and finally have my PS4/PS5 controllers work.

Thanks again @ndeadly for your support.
